src/main/java/com/ea/restaurant/entities/BaseEntity.java

```diff
@@ -0,0 +1,19 @@
+package com.ea.restaurant.entities;
+
+import com.ea.restaurant.constants.Status;
+import java.time.Instant;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@EqualsAndHashCode(callSuper = false)
```

Comment:
You dont need this here. Also add the equals and hascode props

Reply:
Done.